File: 6-Maps-And-Geolocation/Hikers-Watch/app/src/main/java/com/example/hikers_watch/MainActivity.java
package com.example.hikers_watch;

import androidx.appcompat.app.AppCompatActivity;

import android.location.LocationListener;
import android.location.LocationManager;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    LocationManager locationManager;
    LocationListener locationListener;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
}

File: IntentsDemo1/app/src/main/java/com/example/intentsdemo1/MainActivity.kt
package com.example.intentsdemo1

import android.content.Intent
import android.net.Uri
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val edit = findViewById<EditText>(R.id.urlText)
        val btnS = findViewById<Button>(R.id.btnNav);

        btnS.setOnClickListener {
            var url:String = edit.text.toString()
            val intent = Intent (Intent.ACTION_VIEW, Uri.parse(url))
            startActivity(intent)
        }
    }
}